c/src/sbp.c

```diff
@@ -29,27 +29,18 @@
  * ---------
  *
  * First setup a callback for the message you will be receiving. Our callback
- * function must have type #sbp_msg_callback_t or #sbp_frame_callback_t,
- * i.e. it must be of the form:
+ * function must have type #sbp_callback_t i.e. it must be of the form:
```

Comment:
I'm not too aware of the API changes , in the description of this PR it is written that `sbp_msg_callback_t` has been changed to `sbp_callback_t`, what about `sbp_frame_callback_t`? Has that been removed too?